tests/restart/52-cycle-point-time-zone.t

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Test saving of cycle point time zone for restart, which is important for
+# restarting a suite after e.g. a daylight saving change
+
+. "$(dirname "$0")/test_header"
+
+dumpdbtables() {
+    sqlite3 "${SUITE_RUN_DIR}/log/db" \
+        'SELECT * FROM suite_params WHERE key=="cp_tz";' > 'dump.out'
+}
```

Comment:
Should this be moved to flakytests due to sqlite3 usage?

Reply:
sqlite3 is stable.

Tests get put in `flakytests` if they are repeatably intermittent, wait is that a contradiction in terms, hmmm.

Reply:
The first few times I ran the test locally after writing it, it failed due to the database timing out or something, but I guess it seems to be working nowadays

Reply:
This still seems to be flaky for me 😒

Reply:
I've put up a PR to make this test more dependable, Cylc tests suffer from various timing issues which can cause them to become flaky.